src/include/bridge.h

```diff
@@ -12,6 +13,7 @@ void FreeBridge(bridge_t** bridge);
 
 uintptr_t AddBridge(bridge_t* bridge, wrapper_t w, void* fnc, int N, const char* name);
 uintptr_t CheckBridged(bridge_t* bridge, void* fnc);
+bool IsBridge(void* func);
```

Comment:
I know it's nitspicking, but can we avoid the use of `bool`? a simple `int` will do just fine.